-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-35652/25927

ERROR: Build errors found during clang-tidy run.
```
RecoTracker/MkFit/plugins/MkFitEventOfHitsProducer.cc:180:16: error: no member named 'SetBeamSpot' in 'mkfit::EventOfHits' [clang-diagnostic-error]
  eventOfHits->SetBeamSpot(
               ^
RecoTracker/MkFit/plugins/MkFitEventOfHitsProducer.cc:181:14: error: no member named 'BeamSpot' in namespace 'mkfit' [clang-diagnostic-error]
      mkfit::BeamSpot(bs.x0(), bs.y0(), bs.z0(), bs.sigmaZ(), bs.BeamWidthX(), bs.BeamWidthY(), bs.dxdz(), bs.dydz()));
             ^
Suppressed 1399 warnings (1398 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py
+++ b/RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py
@@ -36,6 +36,7 @@ def customizeHLTIter0ToMkFit(process):
         minGoodStripCharge = dict(refToPSet_ = 'HLTSiStripClusterChargeCutLoose'),
     )
     process.hltIter0PFlowCkfTrackCandidatesMkFitEventOfHits = mkFitEventOfHitsProducer_cfi.mkFitEventOfHitsProducer.clone(
+        beamSpot  = "hltOnlineBeamSpot",
         pixelHits = "hltIter0PFlowCkfTrackCandidatesMkFitSiPixelHits",
         stripHits = "hltIter0PFlowCkfTrackCandidatesMkFitSiStripHits",
     )

